pom.xml

```diff
@@ -263,7 +263,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-project</artifactId>
-      <version>1.4.1</version>
+      <version>1.4</version>
```

Comment:
Never mind, I've finally read the Wiki. This Matrix Project thing should have been released as 2.0 at least